build-docs.js

```diff
@@ -0,0 +1,67 @@
+/*
+ * build-docs.js
+ * This program will copy/convert files in /docs/ to /out/docs/
+ */
+/*jslint node*/
+"use strict";
+let directoryOutDocs;
+let fs;
+let head;
+let marked;
+let path;
+let tail;
+// chdir to /docs/
+process.chdir("docs");
+// require modules
+fs = require("fs");
+marked = require("marked");
+path = require("path");
+// init toplevel variables
+directoryOutDocs = path.normalize("../out/docs/0").slice(0, -1);
+head = fs.readFileSync("head.html.part", "utf8");
+tail = fs.readFileSync("tail.html.part", "utf8");
+// process files in /docs/
+fs.readdirSync(".").forEach(function (file) {
+    let data;
+    switch (path.extname(file)) {
+    // copy files *.css, *.html, *.js to /out/docs
+    case ".css":
```

Comment:
it's most common to indent case statements inside switch blocks (to be consistent with every single other occurrence of things inside blocks)
```suggestion
      case ".css":
```

Reply:
jslint forbids indenting case-statement.  eslint has no preference.

fyi, lint-enforcement seems overly lax.  the following passed `npm run lint`:
```js
process.chdir(("docs")); 1
```
had to introduce obvious syntax-error for enforcement to kick-in.

Reply:
jslint hasn't been relevant for the better part of a decade, and isn't used in this repository.

The eslint `indent` rule can be set to enforce this indentation (whether it's enforced here or not), as it is so enforced in all of the common formats (airbnb, standard, prettier).

Reply:
ok follow airbnb styleguide suggestion in follow-on commit bf124c9af

Reply:
FWIW, I have tried to tackle the lint enforcement being overly lax in #447. I discovered we did have a config for 'prettier' already set up, but it wasn't being used.